### PR TITLE
Add Safe contract addresses for version 1.3.0 on Orderly Sepolia testnet

### PR DESCRIPTION
**Pull request by zlog-in**
# ChainId
4460

# Chainlist
https://chainlist.org/chain/4460

# Contract deployed

I confirm that the contracts are deployed

# Version of contracts deployed
v1.3.0

# Relevant information
* RPC endpoint: https://l2-orderly-l2-4460-sepolia-8tc3sd7dvy.t.conduit.xyz
* Block explorer: https://testnet-explorer.orderly.org/

### CODE REVIEW

File: src/assets/v1.3.0/gnosis_safe.json [hunk 1]
```diff
@@ -86,6 +86,7 @@
     "2222": "0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552",
     "3737": "0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552",
     "4002": "0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552",
+    "4460": "0xb4A7C7da1631CF60A2Cf23ABc86986f99a1A7f70",
```
Comment (mmv08) [inline src/assets/v1.3.0/gnosis_safe.json:89]: This address wasn't deployed deterministically. The address should match 1 of 2 addresses in the json file.

**Review by mmv08 — CHANGES_REQUESTED** [review]

File: src/assets/v1.3.0/gnosis_safe.json (hunk 1 above)
Comment (mmv08) [reply src/assets/v1.3.0/gnosis_safe.json:89]: It's not about the mnemonic, it's about using the correct git tag and the deterministic deployment factory (configured by default)

Comment (zlog-in) [reply src/assets/v1.3.0/gnosis_safe.json:89]: Understood, actually I've missed the Singleton factory for this chain, cuz I used the older [deterministic deployment factory](https://github.com/Arachnid/deterministic-deployment-proxy), which provides a presigned tx without chainId included. 
Now I've [submitted an issue](https://github.com/safe-global/safe-singleton-factory/issues/218) to Singleton factory to ask for the presigned tx for this chain from Safe team, then I will redeploy all the Safe contracts.
Thank you.

Comment (mmv08) [reply src/assets/v1.3.0/gnosis_safe.json:89]: There are two addresses in the files: one with the deterministic deployment factory and another with the singleton factory. So, if you used the older one, the addresses still should match one of the 2 in the file.

Comment (zlog-in) [reply src/assets/v1.3.0/gnosis_safe.json:89]: I met a reusing satuation when redeploy the contract with the the deterministic deployment factory,
```
reusing "SimulateTxAccessor" at 0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da
reusing "GnosisSafeProxyFactory" at 0xE89ce3bcD35bA068A9F9d906896D3d03Ad5C30EC
reusing "DefaultCallbackHandler" at 0x1AC114C2099aFAf5261731655Dc6c306bFcd4Dbd
reusing "CompatibilityFallbackHandler" at 0xe16bA5bF81E5BB113e4752E4fdC20351d796fB24
reusing "CreateCall" at 0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4
reusing "MultiSend" at 0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761
reusing "MultiSendCallOnly" at 0x40A2aCCbd92BCA938b02010E17A5b8929b49130D
reusing "SignMessageLib" at 0x03F886722b44BefB13871D4a05621D38616D3b7c
reusing "GnosisSafeL2" at 0xE51abdf814f8854941b9Fe8e3A4F65CAB4e7A4a8
reusing "GnosisSafe" at 0xb4A7C7da1631CF60A2Cf23ABc86986f99a1A7f70
```
How can I solve this to redeploy to get the expected addresses?

Comment (mmv08) [reply src/assets/v1.3.0/gnosis_safe.json:89]: What tag are you using?

Comment (zlog-in) [reply src/assets/v1.3.0/gnosis_safe.json:89]: Actually I've adopted this [Gnosis-Subnet repo](https://github.com/ava-labs/gnosis-subnet), which use this commit of safe contract repo as the submodule: https://github.com/safe-global/safe-contracts/tree/c36bcab46578a442862d043e12a83fec41143dec

It seems to be v1.3.0

Comment (mmv08) [reply src/assets/v1.3.0/gnosis_safe.json:89]: This commit hash doesn't seem to correspond to any of the release tags: https://github.com/safe-global/safe-contracts/releases

Comment (zlog-in) [reply src/assets/v1.3.0/gnosis_safe.json:89]: yes, we used the wrong version. Now we checkouted to the v1.3.0-lib.0, and reployed the contracts with correct address. Will commit soon.
Thank you!

**Pull request closed without merging**